src/SpeedDial/speeddial.st.css

```diff
@@ -0,0 +1,63 @@
+@st-import [--spacing-unit] from "../styles/default/project.st.css";
+@namespace "SpeedDial";
+
+.root {
+  -st-states:
+    speedDialOpen(string);
```

Comment:
Can speedDialOpen be typed as a `boolean`? If it is not possible then maybe we could use `true | false` or `"true" | "false"`?